src/globalTypes.ts

```diff
@@ -0,0 +1,124 @@
+export interface prodData {
```

Comment:
Usually developers use names with I for interfaces and with T for types. E.g. IProdData. Like your Iprops